### DISCUSSION TITLE
About estimator parameters

### DISCUSSION THREAD

**Discussion by mipypf** [pycaret/pycaret#1124] [Q&A] [answered]
Hi All,

I was impressed with the functionality of Pycaret.

Can you help me understand the following questions?

[Questions]
Are the Model Hyperparameters of the various estimators generated by "pycaret.regression.compare_models" and "pycaret.regression.create_model" the default parameters of the referenced library? Also, the parameters displayed in "pycaret.regression.plot_model (estimator, plot =' parameter'" are the parameters explicitly specified on Pycaret, and the Model Hyperparameters that are not displayed are referenced. Can you think of it as the default parameter of the library? For example, "pycaret.regression.plot_model (estimator, plot ='parameter'" has few parameters, and many refer to "Catboost" or None. I am interested in "XGBoost" which seems to be different from the setting value of the default parameter of the library.

Very much appreciate your support

**Comment by Yard1** [comment, marked as answer]
You can find the default parameter overrides here: https://github.com/pycaret/pycaret/blob/master/pycaret/containers/models/regression.py

**Reply by mipypf** [reply]
Thanks for the quick response. 

I saw the above code. In this code, is it okay to recognize that each class refers to the original ML library and adopts the default hyperparameters, while explicitly specifying the hyperparameters on Pycaret in the "args" part? I am studying, so I would appreciate it if you could teach me.

Very much appreciate your support

**Reply by Yard1** [reply]
`args` variables indeed control the default parameters.

**Reply by mipypf** [reply]
Thanks for the quick response.

I get it.

Thank you for your continued support.
